### PR TITLE
Add docstrings for Bio.Phylo

### PR DESCRIPTION
**Pull request by souravsingh**
The PR adds docstrings for a few Bio.Phylo modules and also attempts to fix issues with Bio.Phylo docstrings.

### CODE REVIEW

File: Bio/Phylo/CDAOIO.py [hunk 1]
```diff
@@ -96,6 +99,7 @@ def __init__(self, handle=None):
 
     @classmethod
     def from_string(cls, treetext):
+        """Convert the text to StringIO format."""
```
Comment (etal) [inline Bio/Phylo/CDAOIO.py]: This should be something like: "Instantiate the class from the given string."

File: Bio/Phylo/NeXML.py [hunk 2]
```diff
@@ -16,6 +16,7 @@ class Tree(BaseTree.Tree):
     """NeXML Tree object."""
 
     def __init__(self, root=None, rooted=False, id=None, name=None, weight=1.0):
+        """Initialize paramters for NeXML tree object."""
```
Comment (etal) [inline Bio/Phylo/NeXML.py]: \*parameters, or: "Instantiate a NeXML tree object with the given parameters."

File: Bio/Phylo/Newick.py [hunk 3]
```diff
@@ -25,6 +26,7 @@ class Clade(BaseTree.Clade):
 
     def __init__(self, branch_length=None, name=None, clades=None,
                  confidence=None, comment=None):
+        """Initialize parametrs for a Newick Clade object."""
```
Comment (etal) [inline Bio/Phylo/Newick.py]: \*parameters

File: Bio/Phylo/NewickIO.py [hunk 4]
```diff
@@ -90,10 +90,12 @@ class Parser(object):
     """
 
     def __init__(self, handle):
+        """Initialize file handle for the Newick Tree."""
         self.handle = handle
 
     @classmethod
     def from_string(cls, treetext):
+        """Convert File handle to StringIO obejct."""
```
Comment (etal) [inline Bio/Phylo/NewickIO.py]: As above: "Instantiate the Newick Tree class from the given string."

File: Bio/Phylo/NeXMLIO.py [hunk 5]
```diff
@@ -111,14 +111,17 @@ class Parser(object):
     """
 
     def __init__(self, handle):
+        """Initialize parameters fo NeXML file parser."""
```
Comment (peterjc) [inline Bio/Phylo/NeXMLIO.py]: Typo "fo" --> "for"

**Comment by peterjc** [comment]
I spotted one more typo ("fo" --> "for") and after that this looks good to merge :)

I was going to ask you to review this @etal, but you already have. Please go ahead and squash-and-merge this when that's fixed.

**Pull request merged**